tg.js

```diff
@@ -57,12 +57,12 @@ generic.username Username
 generic.password  Illagay ang password
 generic.register Mag-parehistro
 generic.login Mag log-in
-generic.accuracy Kawastuhan ng pagbaril
+generic.accuracy Accuracy sa paglalaro
 player.waiting Naghihintay ng mga manlalaro
 player.died Ikaw ay namatay
-player.killed {0} Pumatay kay {1}
-player.killstreak {0} nasa {1} Kill Streak
-player.inactive Pina-alis dahil sa walang aktibidad
+player.killed si {0} ay pumatay kay {1}

```

Comment:
Instead of "Si Player A ay pumatay kay Player B", make it "Pinatay ni Player A si Player B" because its a bit long

Reply:
I committed it, with the other suggestions of yours, Thanks.